File: 0070-climbing-stairs/0070-climbing-stairs.java
class Solution {
    public int climbStairs(int n) {
        int res = 0;
        int n_1 = 0;
        int n_2 = 1;

        for(int i = 0; i <= n; i++){
            res = n_1 + n_2;
            n_2 = n_1;
            n_1 = res;    
        }

        return res;
    }
}

File: 0070-climbing-stairs/0070-climbing-stairs.go
func climbStairs(n int) int {
    temp, fn_1, fn_2 :=0, 1, 1

    
    for i:= 2; i <= n ; i++ {
        temp = fn_2
        fn_2 = fn_1 + fn_2
        fn_1 = temp
        
    }

    return fn_2
}